get_new_package.py

```diff
@@ -4,39 +4,75 @@
 import requests
 from sys import exit, argv
 
+# Constants
+###########
 root = Path(__file__).parent
 debian = root.joinpath("debian")
+packages = debian.joinpath("Packages").open().read()
+###########
 
-initial_req = requests.get("https://discord.com/api/download?platform=linux&format=deb", allow_redirects=False)
-initial_req.raise_for_status()
 
-packages = debian.joinpath("Packages").open().read()
+def verify_for_package(json_package_path, url, filename):
+    global packages
+    packages_list = Path(json_package_path)
+    if filename not in packages or (len(argv) > 1 and argv[1] == "--force"):
+        with packages_list.open() as raw_packages:
+            existing_packages = json.load(raw_packages)
+        existing_packages[filename] = url
+        sorted_keys = sorted(existing_packages.keys())
+        while len(existing_packages) > 5:
+            key = sorted_keys[0]
+            del existing_packages[key]
+            sorted_keys = sorted_keys[1:]
+
+        json.dump(existing_packages, packages_list.open("w"), indent=2)
+
+        for package_name, package_url in existing_packages.items():
+            if debian.joinpath("pool", package_name).exists():
+                continue
+            print("getting", package_name)
+            with requests.get(package_url, stream=True) as r:
+                local_path = debian.joinpath("pool", package_name)
+                with local_path.open('wb') as f:
+                    shutil.copyfileobj(r.raw, f)
+        return 0
+    else:
+        print(f"Already have {filename}", url)
+        return 1
```

Comment:
The return 0/1 was an exit code. For this version, it should return a bool from this function and if any of the calls to this return true, return 0 else 1